src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -541,14 +542,24 @@ public void setRestingPlaces(boolean restingPlaces) {
     }
 
     public void setCrowding(double crowding) {
-        if (crowding > 1) {
-            LOG.warn("Invalid crowding value of {} passed in; correcting to 1 (range of 0 to 1)");
+        if ((crowding > 1) || (crowding < -1)) {
```

Comment:
I was thinking to just correct to the default, but that's better, so I'll do that.